commit 8847d477d22fdde37670e906916e0bfeae82e205
Author: Volker Krause <vkrause@kde.org>
Date:   Thu Jul 6 17:16:41 2023 +0200

    Add Gradle proeprties set by Qt
    
    We don't get those as we provide the gradle.properties file ourselves here.
    And without those Gradle fails with OOM with Qt 5.15.10.

diff --git a/src/app/android/gradle.properties b/src/app/android/gradle.properties
index 066ca7ab..fc750e88 100644
--- a/src/app/android/gradle.properties
+++ b/src/app/android/gradle.properties
@@ -1,3 +1,7 @@
 # SPDX-FileCopyrightText: 2020 Gabriel Souza Franco <gabrielfrancosouza@gmail.com>
 # SPDX-License-Identifier: BSD-3-Clause
 android.useAndroidX=true
+
+# Gradlde properties set by Qt
+org.gradle.jvmargs=-Xmx2500m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.parallel=true
